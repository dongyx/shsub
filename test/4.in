<%	notes='note1 <%%note2 note3%%> note4'
%><%	for i in $notes; do%><%=- <%%$i%%>%>
<%	done-%>
