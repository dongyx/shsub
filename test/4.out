- <%note1%>
- <%<%note2%>
- <%note3%>%>
- <%note4%>
